lib/pages/signup/about-page.js

```diff
@@ -43,4 +43,30 @@ export default class AboutPage extends AsyncBaseContainer {
 			await driverHelper.setCheckbox( this.driver, By.css( '#promote' ) );
 		}
 	}
+
+	async unsetCheckBox( {
+		showcase = false,
+		share = false,
+		sell = false,
+		educate = false,
+		promote = false,
+	} ) {
+		// Wait before click and unset checkbox
+		await driverHelper.waitForFieldClearable( this.driver, By.css( '#siteTitle' ) );
+		if ( showcase === true ) {
+			await driverHelper.unsetCheckbox( this.driver, By.css( '#showcase' ) );
+		}
```

Comment:
Could you just pass a selector or a field id and then unset whatever value is set?  This function seems a bit more elaborate than it needs to be.